scripts/activate

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+require 'bundler'
+Bundler.setup(:default, :production)
+
+require 'login_gov/hostdata'
+require 'logger'
+
+logger = Logger.new(STDOUT)
+logger.progname = 'script/activate'
+
+root = File.expand_path('../../', __FILE__)
+logger.info "app root: #{root}"
+
+LoginGov::Hostdata.s3(logger: logger).download_configs(
```

Comment:
I removed the `in_datacenter` block -- just assuming we want to run this and fail loud;y if anything is wrong, but open to putting it back?

Also: I notice we kind of mix `bin/` and `scripts/` (plural, not singular) so I wasn't sure where we wanted to put this, so I went with here in `scripts/`

cc @brodygov